src/main/scala/no/ndla/learningpathapi/validation/LanguageValidator.scala

```diff
@@ -30,7 +32,7 @@ trait LanguageValidator {
 
   object LanguageValidator {
     def validate(fieldPath: String, languageCode: String): String = {
-      languageValidator.validate(fieldPath, languageCode) match {
+      languageValidator.validate(fieldPath, languageCode, allowUnknownLanguage=false) match {
```

Comment:
Vi kan gjerne ta en IRL-walkthrough, jeg er ledig når som helst. Grunnen til at jeg bare defaultet på `false` uten å få lov å overstyre er egentlig bare at validate fra companion objektet bare er i bruk fra tester, ingen andre steder.

Den kan være "nice to have" for senere, men siden den egentlig ikke er i bruk kunne den nok blitt fjernet også..